resources/lib/clientinfo.py

```diff
@@ -39,3 +39,6 @@ def getDeviceId(self):
     def getVersion(self):
         version = xbmcaddon.Addon(id="plugin.video.embycon").getAddonInfo("version")
         return version
+
+    def getClient(self):
+        return 'Kodi EmbyCon'
```

Comment:
![Kodi EmbyCon](http://i.imgur.com/FxDpYdi.png)
Seeing it show as XBMC is what prompted me to look at it. :)
It's showing fine, and no impact on use that I've seen.